recipe/meta.yaml

```diff
@@ -44,24 +30,24 @@ requirements:
     - python
     - blas 1.1 {{ variant }}
     - openblas 0.2.20|0.2.20.*
-    - numpy 1.9.*
+    - numpy 1.13.*
```

Comment:
What's the reason for the new `numpy` version being required?

Reply:
nope. It's just because I currently use 1.13 and want to make sure it works... I will change it back to 1.9.